SuperPong/Assets/Scripts/BallController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections;
```

Comment:
I made a gist of most of the changes that I proposed to figure out what worked.  :)

https://gist.github.com/atb-brown/5c0752f2f391f520de0ba4c76046a6c1